client/lib/utils/time-ago.js

```diff
@@ -0,0 +1,48 @@
+import { translate as __ } from 'lib/mixins/i18n';
+import { sprintf } from 'sprintf-js';
+
```

Comment:
I decided to create my own - from what I read it is difficult to get `moment` to use the correct locale when it's installed with npm, and it seemed that we are not using large parts of that library.